tests/hstestcase.py

```diff
@@ -63,13 +66,24 @@ def _delete_job(cls, jobkey):
 
     @classmethod
     def _set_testbotgroup(cls):
-        cls.project.settings.apipost(jl={'botgroups': [cls.testbotgroup]})
+        for botgroup in cls.testbotgroups:
+            # Settings
+            # endpoint wouldn't accept list > 1
+            cls.project.settings.apipost(jl={'botgroups': [botgroup]})
```

Comment:
Setting more than one botgroup at once returns a `400` status code. However, this is no longer needed after the last changes.